### PR TITLE
Release configs for kubevirt/kubevirt release branch release-1.4

### PR DESCRIPTION
**Pull request by xpivarc**
adds new release configs

### CODE REVIEW

**Review by dhiller — CHANGES_REQUESTED** [review]
@xpivarc the jobs for 1.29 and 1.30 in this config need to get switched to `always_run: true` - you can find these by looking at jobs in the main config that have `run_before_merge: true`

**Comment by brianmcarey** [comment]
@xpivarc @dhiller - I think we need to get this merged in case anyone creates a backport to 1.4.

**Review by brianmcarey — COMMENTED** [review]
Thanks @xpivarc  - just a couple of comments inline

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-1.4.yaml [hunk 1]
```diff
@@ -0,0 +1,1963 @@
...
+    labels:
+      preset-bazel-cache: "true"
+      preset-bazel-unnested: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-podman-in-container-enabled: "true"
+      preset-podman-shared-images: "true"
+      preset-shared-images: "true"
+    max_concurrency: 11
+    name: pull-kubevirt-e2e-k8s-1.29-sig-compute-root-1.4
+    optional: true
```
[the remark this answers is not in the document]
Comment (xpivarc) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-1.4.yaml:1241]: I am not sure about that, do we require root anywhere?

[the remark this answers is not in the document]
Comment (brianmcarey) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-1.4.yaml:1241]: Sorry you are right - early morning

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-1.4.yaml [hunk 2]
```diff
@@ -0,0 +1,1963 @@
...
+        image: quay.io/kubevirtci/bootstrap:v20241014-80f340c
+        name: ""
+        resources:
+          requests:
+            memory: 29Gi
+        securityContext:
+          privileged: true
+      nodeSelector:
+        type: bare-metal-external
+  - always_run: false
```
[the remark this answers is not in the document]
Comment (xpivarc) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-1.4.yaml:1223]: This is root and I am not sure if it is required

**Review by dhiller — COMMENTED** [review]
/approve

/hold to wait for cache drop

**Comment by dhiller** [comment]
> /approve
> 
> /hold to wait for cache drop

Hmm, I am still not seeing the expected change from `always_run: false` to  `always_run: true`, as also @brianmcarey pointed out.

**Review by dhiller — APPROVED** [review]

**Review by brianmcarey — COMMENTED** [review]
/lgtm

/hold cancel

**Pull request merged**